src/Pulp/Publish.purs

```diff
@@ -84,6 +86,13 @@ action = Action \args -> do
               <> " report this: https://github.com/bodil/pulp/issues/new")))
           pure
 
+checkBowerProject :: AffN Unit
+checkBowerProject = do
+  bower <- FS.exists "bower.json"
+  if bower then pure unit
+    else throwError
+         (error "No bower.json found, publishing is currently only implemented for Bower projects.")
```

Comment:
I see your point, but not really finding the words, could you suggest what it should say?